{"name":"Story 991, Test 1205 ; ValidateCallTakeCantCancelCaseWithoutCancellationReason","status":"passed","statusDetails":{"known":false,"muted":false,"flaky":false},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1687965742760,"stop":1687965791834,"uuid":"72ee909f-3430-4b57-9254-3c93fe6d4e82","historyId":"c277972243833ab5285fe82598a6501","fullName":"Execute.CallTakerExecute.ValidateCallTakeCantCancelCaseWithoutCancellationReason","labels":[{"name":"package","value":"Execute.CallTakerExecute"},{"name":"testClass","value":"Execute.CallTakerExecute"},{"name":"testMethod","value":"ValidateCallTakeCantCancelCaseWithoutCancellationReason"},{"name":"parentSuite","value":"Default Suite"},{"name":"suite","value":"BCAA"},{"name":"subSuite","value":"Execute.CallTakerExecute"},{"name":"host","value":"LAP10-GUYL"},{"name":"thread","value":"34428@LAP10-GUYL.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"}],"links":[]}